allvers/2003-12-03-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,5 @@
+Date: 2003-12-03  or earlier
```

Comment:
between October 1, 2003 and December 3, 2003

Reply:
Updated in 6d701cb55db6f4f26097400fa36b5b171c165ef1